open_ce/build_image.py

```diff
@@ -77,21 +77,22 @@ def build_runtime_docker_image(args):
     using locally built conda packages and environment file.
     """
     local_conda_channel = os.path.abspath(args.local_conda_channel)
-    conda_env_file = os.path.abspath(args.conda_env_file)
-    _validate_input_paths(local_conda_channel, conda_env_file)
+    for conda_env_file in parse_arg_list(args.conda_env_files):
+        conda_env_file = os.path.abspath(conda_env_file)
+        _validate_input_paths(local_conda_channel, conda_env_file)
```

Comment:
Here, validate_input_paths is called for as many times as that of conda_env_files and re-validating local_conda_channel every time which is fixed. I think we should change this validate function itself to accept list of conda_env_files and local_conda_channel.